src/commands/junit/api.ts

```diff
@@ -49,17 +55,8 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
 
   form.append('event', JSON.stringify(metadata), {filename: 'event.json'})
 
-  let uniqueFileName = `${fileName}-${jUnitXML.service}-${metadata[GIT_SHA]}`
-
-  if (metadata[CI_PIPELINE_URL]) {
-    uniqueFileName = `${uniqueFileName}-${metadata[CI_PIPELINE_URL]}`
-  }
-  if (metadata[CI_JOB_URL]) {
-    uniqueFileName = `${uniqueFileName}-${metadata[CI_JOB_URL]}`
-  }
-
   form.append('junit_xml_report_file', fs.createReadStream(jUnitXML.xmlPath).pipe(createGzip()), {
-    filename: `${getSafeFileName(uniqueFileName)}.xml.gz`,
+    filename: `${getSafeFileName(fileName)}.xml.gz`,
```

Comment:
let's do `.replace(/\//g, '\\')` here directly. It's 1 function and it's more obvious what it does as it's simpler that it was before.

If you want to keep it in a function rename it to replaceSlashes or similar

Reply:
I would prefer to have it in a separate function even if it is simpler than before. The rename sounds good to me 👍